Changed order of pinecone explaination too avoid confusion (#317)
Torantulino/Auto-GPT: README.md

3. Find your API key and region under the default project in the left sidebar.

### Setting up environment variables

Simply set them in the `.env` file. 

> Alternatively you can use the appropriate following method (advanced):

Reviewer: ```suggestion
Alternatively you can use set them from the command line (advanced):
```

Author: Thanks, that looks better. Can I approve of this some way? Do I need to do something? Sorry not that well versed with GitHub.
